src/Root.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { Route, Routes } from 'react-router-dom';
+import DevTest from './scenes/DevTest';
+import Home from './scenes/Home';
+
+export default () => (
+  <Routes>
+    <Route path="/" element={<Home />} />
+    <Route path="/devtest" element={<DevTest />} />
```

Comment:
What do you think about using `children` instead of `element`? I think it might look cleaner.

Reply:
Good call!